pep-0689.rst

```diff
@@ -130,7 +130,7 @@ Several rules for dealing with the unstable tier will be introduced:
    this name.
 
    In the following cases, an incompatible change (and thus removing the
-   deprecated name) is allowed without an SC exeption, as if the function was
+   deprecated name) is allowed without an SC exception, as if the function was
```

Comment:
```suggestion
   deprecated name) is allowed without an SC exemption, as if the function was
```
Not sure about this one, could be either really and they give a pretty similar meaning, although I think `exemption` may work *slightly* better? Feel free to leave though.

Reply:
The document already uses `exception` once so I took that word:

https://github.com/python/peps/blob/5376172324993c9ec011f24ed64b1eb31f6180c8/pep-0689.rst?plain=1#L124-L125